ustriage/task.py

```diff
@@ -132,6 +133,17 @@ def short_title(self):
         }[self.obj.target.resource_type_link]
         return ' '.join(self.title.split(' ')[start_field:]).replace('"', '')
 
+    def get_flags(self):
+        """Get flags representing the status of the task."""
+        flags = '%s%s' % (
+            '*' if self.subscribed else '',
+            '+' if self.last_activity_ours else '',
+        )
+        if self.AGE:
+            if self.date_last_updated > self.AGE:
+                flags += 'U'
+        return flags
```

Comment:
Kind of a nit: the checks for `U` are different from the checks for `*+`. I'd either go for
```
        flags = '%s%s%s' % (
            '*' if self.subscribed else '',
            '+' if self.last_activity_ours else '',
            'U' if self.AGE and self.date_last_updated else '',
        )
```
or:
```
flags = ''
if self.subscribed: flags += '*'
if self.last_activity_our: flags += '+'
if self.AGE and self.date_last_updated > self.AGE: flags += 'U'
```
I prefer this second way.